src/olympia/stats/tests/test_commands.py

```diff
@@ -45,7 +46,7 @@ def tearDown(self):
         super(FixturesFolderMixin, self).tearDown()
 
 
-class TestADICommand(FixturesFolderMixin, TestCase):
+class TestADICommand(FixturesFolderMixin, TransactionTestCase):
```

Comment:
So, the issue with this is, you need to clear the data between each test yourself. This is not done here, so how the hell are the tests passing ?